plugins/uberdev/commands/turbo.md

```
@@ -51,3 +51,41 @@ bash "$CLAUDE_PLUGIN_ROOT/lib/solve-launcher.sh" --auto-mode=1 --turbo -- $ARGUM
+
+## Workflow mandate (backend=workflow — the default)
+
+When the launcher resolves the **`workflow`** backend (what `auto` selects on
+every Claude host) it does **not** dispatch anything. It stops after the claim
+protocol, writes the per-issue manifest, and prints an args envelope between
+`WORKFLOW_ARGS_BEGIN` / `WORKFLOW_ARGS_END`.
+
+Relay the JSON between those markers **verbatim** (DR-2 — never compose or edit
+the handoff yourself) into:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)